PrebidMobile/ConfigurationAndTargeting/Prebid.swift

```diff
@@ -225,4 +225,12 @@ public class Prebid: NSObject {
     override init() {
         timeoutMillis = defaultTimeoutMillis
     }
+    
+    public static func registerPluginRenderer(_ prebidMobilePluginRenderer: PrebidMobilePluginRenderer) {
```

Comment:
<img width="708" alt="Screenshot 2024-05-16 at 10 53 54" src="https://github.com/prebid/prebid-mobile-ios/assets/99195905/97c925d2-43fe-4450-8ded-a364bcee06cd">

It Supposed to comply with the class diagram, is it missing a function to check if a plugin renderer exists